app/helpers/template_helpers.py

```diff
@@ -73,12 +73,20 @@ def context(self) -> dict[str, Any]:
         }
 
     @property
-    def service_links_context(self) -> Optional[dict[str, list[dict]]]:
+    def service_links_context(
+        self,
+    ) -> Optional[dict[str, Union[dict[str, str], list[dict]]]]:
         if service_links := self._survey_config.get_service_links(
             sign_out_url=self._sign_out_url,
             is_authenticated=current_user.is_authenticated,
         ):
-            return {"itemsList": service_links}
+            return {
+                "toggleServicesButton": {
+                    "text": lazy_gettext("Menu"),
+                    "ariaLabel": lazy_gettext("Toggle services menu"),
```

Comment:
Non blocker: Historically, we have avoided marking aria labels for translations. Prob needs another revisit as we have a mix now, I think mostly unmarked. I think it was driven by whether screen readers etc support non-eng.

Reply:
Will remove the aria label stuff for now and add a card to look into screenreader language support